kyuubi-ctl/src/main/scala/org/apache/kyuubi/ctl/cmd/Command.scala

```diff
@@ -57,11 +57,13 @@ abstract class Command[T](cliConfig: CliConfig) extends Logging {
   private def useDefaultPropertyValueIfMissing(): CliConfig = {
     var arguments: CliConfig = cliConfig.copy()
     if (cliConfig.zkOpts.zkQuorum == null) {
-      conf.getOption(HA_ADDRESSES.key).foreach { v =>
```

Comment:
maybe it can be simply fixed by `conf.get(HA_ADDRESSES)`

Reply:
coding style consistent, done

Reply:
Have you try my suggestion? The `KyuubiConf` system should support fallback mechanism

Reply:
for code style, it can be quickly fixed by `dev/reformat`

Reply:
right `conf.get` is simpler and easy. I didn't notice `.get` and `getOption` have different implementation, thought fallback not working